gulpfile.js

```diff
@@ -135,6 +140,12 @@ gulp.task('build:chrome', ['compress:chrome']);
 
 gulp.task('build:firefox', ['compress:firefox']);
 
+gulp.task('build:extensions', ['lint:scripts'], function(callback) {
```

Comment:
Yes, Extensions/dist/*.json and Extensions/dist/{gallery,list}.json should probably be cleaned